pkg/api/schema/schema.proto

```diff
@@ -345,6 +345,9 @@ message ImmutableState {
 	uint64 txId = 2;
 	bytes txHash = 3;
 	Signature signature = 4;
+	// following fields are not part of the signature
+	uint64 precommittedTxId = 5;
```

Comment:
Why those have to re reported ?

Reply:
as discussed offline, that information may be needed to decide which follower may become the new master